### PR TITLE
ParticleNet: Dynamic Batching and PseudoRagged Batching

### PR DESCRIPTION
**Pull request by wpmccormack**
Adding dynamic batching settings to ParticleNet and making the number of particles and vertices expected per jet in an event a variable

### CODE REVIEW

File: models/particlenet/config.pbtxt [hunk 1]
```diff
@@ -1,36 +1,39 @@
 name: "particlenet"
 platform: "onnxruntime_onnx"
-max_batch_size : 16
+max_batch_size : 160
+dynamic_batching {
+   preferred_batch_size: [ 80 ]
```
Comment (kpedro88) [inline models/particlenet/config.pbtxt:5]: has this choice been optimized?

File: models/particlenet_AK4/config.pbtxt [hunk 2]
```diff
@@ -1,38 +1,38 @@
 name: "particlenet_AK4"
 platform: "onnxruntime_onnx"
-max_batch_size : 50
+max_batch_size : 500
 dynamic_batching {
```
Comment (kpedro88) [inline models/particlenet_AK4/config.pbtxt:4]: should there be a preferred batch size here? (and in the other configs below)

File: models/particlenet/config.pbtxt (hunk 1 above)
Comment (wpmccormack) [reply models/particlenet/config.pbtxt:5]: I based this on benchmarking results here: https://docs.google.com/presentation/d/1XOhrUFJN6dU_he8doJQz8cmrTgOd8NeQQ6t1PgVJE7s/edit?usp=sharing.  Throughput seems to plateau for batch sizes larger than ~80

File: models/particlenet_AK4/config.pbtxt (hunk 2 above)
Comment (wpmccormack) [reply models/particlenet_AK4/config.pbtxt:4]: I changed these in the latest push.  Also based on the benchmarking from the slides referenced above

**Pull request merged**